docs/docs/detecting/golang.md

```diff
@@ -70,7 +70,7 @@ There are 2 ways you can get your own golang signatures working with tracee.
         
         		switch e.EventName {
         		case "openat", "execve":
-        			arg, err := helpers.GetTraceeArgumentByName(e, "pathname")
+                    arg, err := helpers.GetTraceeArgumentByName(e, "pathname", helpers.GetArgOps{DefaultArgs: false})
```

Comment:
Indentation looks wrong, but we can fix this later I guess. Is it rendered ok and just showing wrong here?